### PR TITLE
Show graph data date if available under "Info"

### PR DESCRIPTION
**Pull request by ratrun**
I'm missing the graph data date information in the new maps. Therefore I created this PR which shows the OSM data date if availailable and as a fallback the import date. Besides that the graphhopper back-end version is displayed.

### CODE REVIEW

**Comment by ratrun** [comment]
Maybe we could put the data/import date into the UI and log the backend version? I believe that the data date is important information for most users, while the exact version is only interesting to developers.

File: src/sidebar/search/Search.tsx [hunk 1]
```diff
@@ -78,13 +79,24 @@ export default function Search({ points }: { points: QueryPoint[] }) {
...
             {showInfo && (
+                <>
+                <div className={styles.dataVersionLine}>
+                    {(getApiInfoStore().state.data_date ==='') && (
+                       <div>Import from {getApiInfoStore().state.import_date} </div>
+                    )}
+                    {(getApiInfoStore().state.data_date !=='') && (
+                       <div>Graph data from {getApiInfoStore().state.data_date} </div>
+                    )}
+                    <div>Version {getApiInfoStore().state.version} </div>
```
[the remark this answers is not in the document]
Comment (ratrun) [reply src/sidebar/search/Search.tsx]: Thanks, I implemented the suggestion here: [e1ab38d](https://github.com/graphhopper/graphhopper-maps/pull/284/commits/e1ab38db991c2da51fcdce94fd9293f6d3d3e674)

**Comment by ratrun** [comment]
> The problem about the data/import date is that it is not a single value. Behind the scene of our API we have now 3 different clusters and so there are 3 different values for this and a single value is misleading.

If I understood this correctly then the data dates would probably need to be moved from the `/info` endpoint to the route calculation response. Is this what you mean?

**Comment by otbutz** [comment]
@karussell maybe you could you set the backend information as a header to include it? e.g `X-Backend: node1.graphhoper.com`

**Comment by ratrun** [comment]
Closed in favour of [https://github.com/graphhopper/graphhopper/issues/2806](https://github.com/graphhopper/graphhopper/issues/2806)

**Pull request closed without merging**